### ISSUE TITLE
webclient is deprecated

### ISSUE THREAD

**Issue by Vincenzoferrara** [ravibpatel/AutoUpdater.NET#676]
https://learn.microsoft.com/en-us/dotnet/core/compatibility/networking/6.0/webrequest-deprecated

**Comment by ravibpatel** [comment]
Yes, I am aware of that, but currently there is no official implementation of FTP support and due to limited time I have it is not possible to upgrade to HttpClient right now. I will upgrade to it in future when I have some more free time.
